src/tutorials-democracy.md

```diff
@@ -0,0 +1,157 @@
+### Democracy Tutorial
+
+In this tutorial, we show a basic flow for two proposals in the democracy module using the CLI. On proposal will be approved and the other one will be rejected.
+
+Go to the encointer-node repo and open 2 terminals:
+
+Terminal 1:
+
+```
+git clone https://github.com/encointer/encointer-node.git
+cd encointer-node && git fetch && git checkout origin/personhood-democracy-1-delivery
+cd ..
+git clone https://github.com/encointer/pallets.git
+mv pallets/ encointer-pallets
+cd encointer-pallets && git fetch && git checkout origin/polkadot-v1.0.0-pallets-v1.3.0-democracy
+cd ../encointer-node
+
+cargo build --release
+./target/release/encointer-node-notee --dev --enable-offchain-indexing true -lencointer=debug,parity_ws=warn
+```
+
+Terminal 2:
+
+```
+cd client
+python -m venv env
+source env/bin.activate
+pip install requirements.txt
+python bootstrap_demo_community.py
+```
+
+Continuing in Terminal 2, we first move to the start of the next cycle, in order for the reputations of the last cycle to become eligible for voting:
+
+```
+../target/release/encointer-client-notee next-phase
+../target/release/encointer-client-notee next-phase
+../target/release/encointer-client-notee next-phase
+../target/release/encointer-client-notee listen -b 1
+```
+
+After that, we submit two porposals for voting:
+
+```
+# Submitting proposal id 1, SetInactivityTimeout(8)
+../target/release/encointer-client-notee submit-set-inactivity-timeout-proposal //Alice 8
+../target/release/encointer-client-notee listen -b 1
+# Submitting proposal id 2, UpdateNominalIncome(cid, 44)
+../target/release/encointer-client-notee --cid sqm1v79dF6b submit-update-nominal-income-proposal //Alice 44
+```
+
+We wait for the extrinsics to be in a block and list the proposals:
+
+```
+../target/release/encointer-client-notee listen -b 1
+../target/release/encointer-client-notee list-proposals
+```
+
+We expect to see the following output:
+
+```
+id: 1
+state: ProposalState::Ongoing
+action: ProposalAction::SetInactivityTimeout(8)
+start time: 1704685908002
+start cindex: 3
+electorate size: 3
+turnout: 0
+ayes: 0
+id: 2
+state: ProposalState::Ongoing
+action: ProposalAction::UpdateNominalIncome(sqm1v79dF6b, 44)
+start time: 1704685920000
+start cindex: 3
+electorate size: 3
+turnout: 0
+ayes: 0
+```
+
+As an exmaple we show that Alice has 1 reputation and therefore a voting power of 1:
+```
+../target/release/encointer-client-notee reputation //Alice
+```
+which will yield:
+```
+1, sqm1v79dF6b, Reputation::VerifiedUnlinked
+```
+The same will hold for Bob and Charlie.  
+
+
+Then, we let the users vote for the porposals:
+
+```
+# Alice votes aye for proposal 1
```

Comment:
I would suggest to add another vote by Alice just to show that her vote only counts once